secureComputeClient/api.go

```diff
@@ -1 +1,78 @@
 package secureComputeClient
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+// SubscriptionManagers wraps the Manager object for a subscription
+type SubscriptionManagers struct {
+	ClientID uuid.UUID `json:"client_id,omitempty"`
```

Comment:
Maybe `ToznyClientID`? Unless this is a separate ID from user or tozny

Reply:
fixing!